systemtests/util_test.go

```diff
@@ -392,7 +392,7 @@ func waitForAPIServer(node remotessh.TestbedNode) error {
 
 func waitForVolplugin(node remotessh.TestbedNode) error {
 	log.Infof("Checking if volplugin is running on %q", node.GetName())
-	err := runCommandUntilNoError(node, "pgrep -c volplugin", 10)
+	err := runCommandUntilNoError(node, "docker ps | grep volplugin | grep -v volsupervisor | grep -v apiserver", 10)
```

Comment:
`docker inspect volplugin` does not really guarantee that the container is `running`.
Shall I use the below instead,
`docker inspect -f {{.State.Running}} volsupervisor | grep true` and check the exit code.